app/models/course_enrichment.rb

```diff
@@ -38,7 +38,7 @@ class CourseEnrichment < ApplicationRecord
   belongs_to :course
 
   scope :latest_first, -> { order(created_at: :desc, id: :desc) }
-  scope :draft, -> { where(status: 'draft').or(CourseEnrichment.rolled_over) }
+  scope :draft, -> { where(status: 'draft').or(CourseEnrichment.unscoped.rolled_over) }
```

Comment:
Why this? If we implement course enrichment this will pull them in too, which might be a tricky bug to track down.

Reply:
This is due to https://github.com/rails/rails/pull/35280/files#diff-2ea4047e83f40a075015865a491c82e7R31

My understanding is: Rails 6.0 and under treat the original line as:

```ruby
scope :draft, -> { where(status: 'draft').or(CourseEnrichment.draft.rolled_over) }
                                                              ^^^^^
```

Rails 6.1+ will treat it as:

```ruby
scope :draft, -> { where(status: 'draft').or(CourseEnrichment.unscoped.rolled_over) }
                                                              ^^^^^^^^
```

With that in mind, I think we should actually be adding the `draft` scope in explicitly (instead of `unscoped`) to maintain current behaviour. So I'll update this.

Reply:
Nevermind, that brings in the Rails deprecation warning:

```
DEPRECATION WARNING: Class level methods will no longer inherit scoping from `draft` in Rails 6.1. To continue using the scoped relation, pass it into the block directly. To instead access the full set of models, as Rails 6.1 will, use `CourseEnrichment.unscoped`. (called from block in <class:CourseEnrichment> at /Users/tvararu/dfe/manage-courses-backend/app/models/course_enrichment.rb:41)
```

It's doing my head in what to do here but all the tests pass with `.unscoped` ?_?